src/unix/linux-core.c

```diff
@@ -1034,7 +1253,10 @@ static uint64_t uv__read_cgroups_uint64(const char* cgroup, const char* param) {
 
   if (n > 0) {
     buf[n] = '\0';
-    sscanf(buf, "%" PRIu64, &rc);
+    if (!strcmp(buf, CGROUPS_V2_NO_LIMIT_VALUE))
+      rc = 0;
```

Comment:
`rc` is initialized to 0 a few lines up, so this `if-else` can probably be rewritten as just an `if (0 != strcmp(buf, CGROUPS_V2_NO_LIMIT_VALUE))`